meson.build

```diff
@@ -13,41 +13,40 @@ project('bmcweb', 'cpp',
             'werror=true',
            ])
 
-# Project related links
+#Project related links
```

Comment:
What is going on here?

Reply:
i ran clang-tidy on this file and found these changes.
(clang-format-17 -i meson.build)

Reply:
This is a meson file. This doesn't match with what is upstream. 
https://github.com/openbmc/bmcweb/blob/master/meson.build#L386

Reply:
Can you revert your changes to this file and only add the one line you need?

Reply:
okay